src/backends/interflop-vprec/interflop_vprec.c

```diff
@@ -455,6 +457,96 @@ static inline double _vprec_binary64_binary_op(double a, double b,
   return res;
 }
 
+// Round the float with the given precision
+static float _vprec_round_binary32( float a,
```

Comment:
There seems to be a lot in common with _vprec_binary32_binary_op.
There should be some factoring here.

Reply:
_vprec_round_binaryXX functions are now used in _vprec_binaryXX_binary_op to round inputs and outputs